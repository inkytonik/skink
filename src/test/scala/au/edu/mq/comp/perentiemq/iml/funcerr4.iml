// Bad argument type
def funcerr4(a : Foo) : (b) {
    a = 1;
}
