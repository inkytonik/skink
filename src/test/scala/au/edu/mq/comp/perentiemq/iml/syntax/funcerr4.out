src/test/scala/au/edu/mq/comp/perentiemq/iml/syntax/funcerr4.iml:3:1: error: non block statement expected
}
^

